tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleClientSuite.scala

```diff
@@ -150,7 +152,8 @@ class RapidsShuffleClientSuite extends RapidsShuffleTestHelper {
       // the transport will receive no pending requests (for buffers) for queuing
       verify(mockTransport, times(0)).queuePending(any())
 
-      assert(response.isClosed)
+      // this is not called since a message implies success
+      verify(mockTransaction, times(0)).releaseMessage()
 
       newMocks()
```

Comment:
```suggestion
```

Reply:
Ah, nevermind, this is one of the cases where it's needed.  However a cleaner way to do this is to either make this a parameterized test or factor out the common code and make multiple tests for the multiple cases.  Bonus points: when the test fails for a particular setting, the test failure lets you know which case it was.  As-is, it will just say the test failed and it's left as an exercise to debug which one.  Not mustfix for this PR.

Reply:
done.